addOns/postman/src/main/java/org/zaproxy/addon/postman/Models/PostmanCollection.java

```diff
@@ -17,21 +17,26 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.zaproxy.addon.postman;
+package org.zaproxy.addon.postman.Models;
```

Comment:
Should be lower case.

Reply:
The directory still seems to be upper case.

Reply:
I renamed that but don't know why it was not tracked by git. Worked by moving the files to a temp folder and then renaming that.

Reply:
I believe that happens because Windows file system is case insensitive.